CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.10.0 (TBD)
+
+### Behavior change
+
+- Removed `date_format` as an alias of `to_date` and use it to format columns based on specified date format.
```

Comment:
I wonder if we should highlight the behavior change as compared to the implementation. For example, how about 
```
- Changed the behavior of `date_format`:
  - The format argument changed from optional to required
  - The returned result changed from a date object to a date-formatted string
```